uberon_edit.obo

```diff
@@ -191394,6 +191396,17 @@ xref: FMA:16029
 intersection_of: UBERON:0000353 ! parenchyma
 intersection_of: part_of UBERON:0002106 ! spleen
 
+[Term]
+id: UBERON:1100000
+name: digestive tract junction
+def: "An anatomical junction between two parts of the digestive tract." [ORCID:0000-0002-7073-9172]
+synonym: "junction of intestine" EXACT []
```

Comment:
not sure if it makes sense to have this as a synonym -- doesn't "digestive tract" include non-intestine parts as well (e.g. https://www.cancer.gov/publications/dictionaries/cancer-terms/def/digestive-tract )

Reply:
@mkeays 
@dosumis 
"junction of intestine" should be a narrow synonym, not an exact one please.

Reply:
(And you may want to add "intestinal junction" also as narrow synonym if not there already)

Reply:
I'll remove.  I think it's there by accident because I clone a term to make this one.